index.html

```diff
@@ -125,5 +125,190 @@ <h2>I'am a Software Developer</h2>
+              has been the industry's standard
+            </p>
+            <div class="inner-button">
+              <button type="button">html</button>
+              <button type="button">bootstrap</button>
+              <button type="button">Ruby</button>
+            </div>
+          </div>
+          <button type="button" class="inner-work-btn">see projects</button>
+        </div>
+
+        <div class="inner-work">
+          <div class="inner-work-text">
+            <h3>professional art <br />printing data</h3>
+            <p>
+              A daily selection of privately personalized reads; no accounts or sign-ups required.
+              has been the industry's standard
+            </p>
+            <div class="inner-button">
+              <button type="button">html</button>
+              <button type="button">bootstrap</button>
+              <button type="button">Ruby</button>
+            </div>
+          </div>
+          <button type="button" class="inner-work-btn">see projects</button>
+        </div>
+
+        <div class="inner-work">
+          <div class="inner-work-text">
+            <h3>professional art <br />printing data</h3>
+            <p>
+              A daily selection of privately personalized reads; no accounts or sign-ups required.
+              has been the industry's standard
+            </p>
+            <div class="inner-button">
+              <button type="button">html</button>
+              <button type="button">bootstrap</button>
+              <button type="button">Ruby</button>
+            </div>
+          </div>
+          <button type="button" class="inner-work-btn">see projects</button>
+        </div>
+
+        <div class="inner-work">
+          <div class="inner-work-text">
+            <h3>professional art <br />printing data</h3>
+            <p>
+              A daily selection of privately personalized reads; no accounts or sign-ups required.
+              has been the industry's standard
+            </p>
+            <div class="inner-button">
+              <button type="button">html</button>
+              <button type="button">bootstrap</button>
+              <button type="button">Ruby</button>
+            </div>
+          </div>
+          <button type="button" class="inner-work-btn">see projects</button>
+        </div>
+      </div>
+    </section>
+    <!-- end of work section -->
+
+    <!-- start of about section -->
+    <section id="about" class="about">
+      <div class="about-div">
+        <h2>About me</h2>
+        <p>
+          Hello I’m a software developer! I can help you build a product , feature or website Look
+          through some of my work and experience! If you like what you see and have a project you
+          need coded, don’t hestiate to contact me.
+        </p>
+        <button type="button">get my resume</button>
+      </div>
+
+      <!-- languages and frameworks part -->
+      <div class="skills">
```

Comment:
We should ensure The technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li` and The technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`